andela/ah-fulldeck-frontend
# #162948962 implement likes and dislikes

#### What does this PR do?
* enables a user to like or dislike an article.

#### Description of Task to be completed?
* Lets appreciate those authors who go out of their way to create good content with a Like and encourage those authors who do not meet our expectations with a dislike.

#### How should this be manually tested?
* git clone this repo

```
$ git clone https://github.com/andela/ah-fulldeck-frontend
```
* Navigate to this branch

```
git checkout to ft-like-dislike-articles-162948962
```
* Install the node modules

```
$ npm install
```
* Start the server

```
$ npm start
```
* It should open your browser to localhost port 3000.
* Log in via the login page then navigate to a single article.
* You should find two icons for like and dislike.
* Use them and see if they work as expected.

#### What are the relevant pivotal tracker stories?
[Users should be able to like and or dislike articles](https://www.pivotaltracker.com/story/show/162948962)
#### Screenshots

##### Authenticated users are able to like or dislike an article and view total likes or dislikes
<img width="1178" alt="screenshot 2019-02-21 at 12 28 29" src="https://user-images.githubusercontent.com/22760022/53158413-8d365d80-35d4-11e9-8769-7b51a3f87ff0.png">


##### Unauthenticated users are able to view the total likes or dislikes
<img width="1184" alt="screenshot 2019-02-21 at 01 25 00" src="https://user-images.githubusercontent.com/22760022/53130152-8549cf80-357a-11e9-8dd5-181d941e67d9.png">
